eslint.config.cjs

```diff
@@ -191,7 +191,7 @@ const config = [{
     },
 
 }, {
-    files: ["**/*.esm.js"],
```

Comment:
These files are synchronized from time to time from the https://github.com/oca/oca-addons-repo-template
This change will then be lost..

I recommend to simply use the `.test.esm.js` suffix on your filename, then it'd be selected by this existing pattern